yarn.lock

```diff
@@ -5606,6 +5606,18 @@ hash.js@^1.0.0, hash.js@^1.0.3:
     inherits "^2.0.3"
     minimalistic-assert "^1.0.1"
 
+history@^4.10.1:
```

Comment:
try to use `BrowserRouter` from `react-router-dom` instead of playing with this package.

Reply:
yes, I had a <BrowserRouter> there but when I needed to history.push in file index.js I started to have some problems and I found this workaround to change it to <Router> and install this 'history' package. I remember that I spend some frustrating time on this, however if you can give me a quick tip how to get back to <BrowserRouter> and avoid issues with history.push as I mentioned - I will be more than grateful.

Reply:
hm. Ok, this is not wrong solution you created, without some kind of pairing I am not able to give you a hint 🤔 Let's stay as is for now